### PR TITLE
 DCOS-40640: reuse Cypress cache

### PR DESCRIPTION
**Pull request by nLight**
With this PR we will be re-using cached Cypress installation that should reduce the build time.

Closes DCOS-40640

## Testing

CI should pass. 
Go inside the build and make sure we're not installing cypress for every set of tests.

This is what should not be present in the logs
screenshot 2018-10-01 at 18 17 48

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -32,6 +32,7 @@ pipeline {
     stage("Build") {
       steps {
         sh "npm --unsafe-perm install"
```
Comment (brandonc) [inline Jenkinsfile]: ❔I thought `cypress install` was already happening postinstall?

File: system-tests/_scripts/sandbox-setup.sh [hunk 2]
```diff
@@ -4,4 +4,6 @@ SCRIPT=`realpath $0`
 SCRIPT_PATH=`dirname $SCRIPT`
 CYPRESS=`realpath "${SCRIPT_PATH}/../../node_modules/cypress/bin/cypress"`
 
-ln -s $CYPRESS "$TMPDIR/cypress"
\ No newline at end of file
+cp -R "/root/.cache" "$TMPDIR/../"
```
Comment (brandonc) [inline system-tests/_scripts/sandbox-setup.sh]: ❔ What manages `/root/.cache`? Jenkins? Does this script work in local dev environment?

**Comment by GeorgiSTodorov** [comment]
@nLight I am leaving this unmerged in case you want to address Brandon's comments.

File: Jenkinsfile (hunk 1 above)
Comment (nLight) [reply Jenkinsfile]: Fair, let's remove it. Thanks for pointing this!

File: system-tests/_scripts/sandbox-setup.sh (hunk 2 above)
Comment (nLight) [reply system-tests/_scripts/sandbox-setup.sh]: The script is supposed to be ran in docker even in local dev environment. So the user is `root`.

Comment (DanielMSchmidt) [reply system-tests/_scripts/sandbox-setup.sh]: I would normally say let's just make it configurable, but the sandbox setup within system tests is pretty specific to being run on docker

**Pull request merged**